### PR TITLE
My Login Page

### PR DESCRIPTION
**Pull request by Martincu-Petru**
Everything looks great and is responsive

### CODE REVIEW

File: TheCrickets/views/login/loginStyleSheet.css [hunk 1]
```diff
@@ -119,6 +175,102 @@ body {
...
+	}
+	/* dfsjfgdfjgifjgijjjjjgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg*/
+	#flooding, #tornado, #earthquake, #fire { 
+		height: 30px !important;
+		width: 30px !important;
+		padding-left: 20px !important;
+	}
+	
+	#floodingText, #fireText, #earthquakeText, #tornadoText {
+		font-size: 20px !important;
```
Comment (CovaliuLucian) [inline TheCrickets/views/login/loginStyleSheet.css]: !important: scoate !important

**Review by CovaliuLucian — CHANGES_REQUESTED** [review]

File: TheCrickets/views/login/loginStyleSheet.css (hunk 1 above)
Comment (Martincu-Petru) [reply TheCrickets/views/login/loginStyleSheet.css]: Aaaaaa, dar o să strice chestii

Comment (Martincu-Petru) [reply TheCrickets/views/login/loginStyleSheet.css]: Am încerat

Comment (CovaliuLucian) [reply TheCrickets/views/login/loginStyleSheet.css]: Incearca mai tare ;)

**Review by CovaliuLucian — APPROVED** [review]
That's better

**Pull request merged**